text_generation/causal_lm/cpp/continuous_batching/library/include/generation_config.hpp

```diff
@@ -38,9 +38,10 @@ struct GenerationConfig {
 
     // Multinomial
     float temperature = 0.0f; // by default we use greedy sampling
-    int top_k = -1; // maybe to assign vocab_size ?
+    int top_k = 0; // HF transformers uses a value of 0 or `None` to disable top-K logit warping
     float top_p = 1.0f; // by default convsider all tokens
     bool do_sample;
```

Comment:
should we assign `false` as default value?

Reply:
Done